ReSharper.FSharp/test/data/features/quickFixes/ignoreUnusedBindingExpression/Multiline 01 - Seq expr.fs

```diff
@@ -0,0 +1,8 @@
+module Module
+
+let a =
```

Comment:
I think we could save up to a half of the dump by using `do` instead of top-level binding.
```fsharp
module Module

do
  let a =
    1
    1

  ()
```

A bonus point would be replacing the top-level module with a anon module, but it'd require more changes in test project options, so I'd rather do it separately.